packages/account/src/Types/common.type.ts

```diff
@@ -162,3 +162,23 @@ export type TServerError = {
     details?: { [key: string]: string };
     fields?: string[];
 };
+
+export type TFormData = {
```

Comment:
Please check if a type from @deriv/api-types can be used. What do u think @amina-deriv?

Reply:
yea.. should be able to use GetFinancialAssessment types. @sanjam-deriv can we check this?

Reply:
Done